pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/UserCreationResponse.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.ddp.json;
+
+import org.broadinstitute.ddp.model.user.User;
+import org.broadinstitute.ddp.model.user.UserProfile;
+
+import com.google.gson.annotations.SerializedName;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+public class UserCreationResponse {
+
+    @NonNull
+    @SerializedName("guid")
+    private String guid;
+
+    @NonNull
+    @SerializedName("hruid")
+    private String hruid;
+
+    @NonNull
+    @SerializedName("email")
+    private String email;
+    
+    @SerializedName("profile")
+    private Profile profile;
+
+    @SerializedName("center")
+    private String center;
+    
+    public UserCreationResponse(User user, UserProfile profile) {
+        this.guid = user.getGuid();
+        this.hruid = user.getHruid();
+        this.email = user.getEmail().orElseThrow(() -> new IllegalStateException("participant must have a valid email"));
```

Comment:
We're saying **valid** here, but we don't check the e-mail to be valid. Just not null. Isn't it a bit misleading?

Reply:
Nope. Valid in this case is "non-null". I'd be open to changing the text of the exception to make it clearer, but it is not this object's responsibility to validate a User's email. If a User was created with an invalid email, that's a bug elsewhere.